extension/encoding/macosunifiedloggingencodingextension/firehose_loss.go

```diff
@@ -1,7 +1,23 @@
 package macosunifiedloggingencodingextension
 
-type FirehoseLoss struct{}
+import "encoding/binary"
+
+type FirehoseLoss struct {
+	StartTime uint64
+	EndTime   uint64
```

Comment:
You think its valuable to put these into a time.TIme? I know the rust implementation does not do that but its hard to tell if this UnixEpoch or not. The test case seems to target a time in 1992?

Tuesday, June 2, 1992 8:58:48 AM?

Maybe a comment on the uint64 value could be useful